### ISSUE TITLE
Implement bc-dao subgraph

### ISSUE THREAD

**Issue by orishim** [dOrgTech/bounties#7]

**Comment by tspoff** [comment]
@dOrgJelli Are you interested in this? I can take it if you're focused on other things.

**Comment by dOrgJelli** [comment]
Yes please feel free to hop on this @tspoff, as I don't think I'll have time in the next 2 weeks.
